phoenix-queryserver/src/main/java/org/apache/phoenix/queryserver/metrics/PqsMetricsSystem.java

```diff
@@ -0,0 +1,105 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.phoenix.queryserver.metrics;
+
+
+import org.apache.hadoop.conf.Configuration;
+import org.apache.phoenix.query.QueryServicesOptions;
+import org.apache.phoenix.queryserver.metrics.sink.PqsFileSink;
+import org.apache.phoenix.queryserver.metrics.sink.PqsSink;
+import org.apache.phoenix.queryserver.metrics.sink.PqsSlf4jSink;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import static org.apache.phoenix.query.QueryServices.PHOENIX_PQS_FILE_SINK_FILENAME;
+import static org.apache.phoenix.query.QueryServices.PHOENIX_PQS_METRIC_REPORTING_INTERVAL_MS;
+import static org.apache.phoenix.query.QueryServices.PHOENIX_PQS_TYPE_OF_SINK;
+
+
+public class PqsMetricsSystem {
+
+    public final static String statementReadMetrics = "Statement-RequestReadMetrics";
+    public final static String overAllReadRequestMetrics = "Statement-OverAllReadRequestMetrics";
+    public final static String connectionWriteMetricsForMutations = "Connection-WriteMetricsForMutations";
+    public final static String connectionReadMetricsForMutations = "Connection-ReadMetricsForMutations";
```

Comment:
These names leave a bit to be desired for me. Reading them, I don't really know what information they're going to contain. How about instead..

* QueryReadMetrics
* GlobalQueryReadMetrics
* ConnectionMutationWriteMetrics
* ConnectionMutationReadMetrics

I don't have a good understanding of what these metrics contain; it just seemed like you copied the names from the internal Phoenix methods. If we're exposing them to users, they need to be self-explanatory (or have documentation explaining them -- maybe we have that?).